modules/wallet/persist.go

```diff
@@ -108,12 +90,39 @@ func (w *Wallet) load() error {
 		}
 	}
 
-	// Load the siafunds file, which is intentionally called 'outputs.dat'.
-	var siafundAddresses []types.UnlockHash
-	err = encoding.ReadFile(filepath.Join(w.saveDir, "outputs.dat"), &siafundAddresses)
+	return nil
+}
+
+// load reads the contents of a wallet from a file.
+func (w *Wallet) load() error {
+	var savedKeys []savedKey
+	err := encoding.ReadFile(filepath.Join(persist.HomeFolder, modules.WalletDir, "wallet.dat"), &savedKeys)
```

Comment:
it's a toss up. You don't want to load the backup first because it would confuse users who replace their home folder's wallet.dat and expect it to be loaded. On the other hand, if you load the home folder first, then when you plug your Sia USB stick into someone else's computer, you'll get their wallet instead of the one on the stick.